tests/cli/output/66.txt

```diff
@@ -224,6 +224,7 @@ log_rule(newlines_cleanup_braces): rule is 'nl_type_brace_init_lst_open'
 log_rule(newlines_cleanup_braces): rule is 'nl_type_brace_init_lst_close'
 log_rule(newlines_brace_pair): rule is 'nl_define_macro'
 log_rule(newlines_brace_pair): rule is 'nl_create_func_def_one_liner'
+log_rule(newlines_brace_pair): rule is 'nl_ctor_brace_cond'
```

Comment:
This will break due to #3038. Please rebase onto latest master and re-run `cmake --build . --target update-cli-options`.

Reply:
...and in fact, I see now it won't even merge cleanly :slightly_smiling_face:.

When rebasing, just `git add tests/cli/output/66.txt && git rebase --continue` without bothering about the conflicts, since that file will be recreated anyway. After doing so, `git commit --amend` your commit. (You might even consider squashing this to a single commit.) Thanks!